lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -120,7 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $query->setMaxResults((int) $maxResult);
         }
 
-        if ($input->hasOption('show-sql')) {
+        if ($input->hasOption('show-sql') && $input->getOption('show-sql')) {
```

Comment:
For the same reason as someone adds when: they think it is (un)necessary to have it. Or in another case, when someone might rename it. 
I do believe this is the cleaner solution, but if you prefer it otherwise, I can change the PR.